src/main/java/mgtow/storage/Storage.java

```diff
@@ -0,0 +1,58 @@
+package mgtow.storage;
+
+import mgtow.task.Task;
+import mgtow.util.InvalidTaskException;
+import mgtow.util.Parser;
+
+import java.io.*;
```

Comment:
Perhaps we can consider importing each method explicitly to keep the amount of code imported to a minimum?